docs/public-documentation/Output.md

```diff
@@ -83,18 +83,22 @@ Here&#39;s the binding data in the _function.json_ file:
 
 \&lt;-- placeholder for the examples --\&gt;
 
+| **function.json property** | **Description** |
+|-|-|
+|type|Must be set to kafkaOutput.|
+|direction|Must be set to out.|
+|name|Name of the variable that represents  request or request body in the function code.|
+|brokerList|Defines the broker list.|
+
 For connection to a secure Kafka Broker -
 
 | **function.json property** | **librdkafka property** | **Description** |
-| --- | --- | --- |
+|-|-|-|
 | authenticationMode | sasl.mechanism | SASL mechanism to use for authentication |
-| --- | --- | --- |
 | username | sasl.username | SASL username for use with the PLAIN and SASL-SCRAM |
 | password | sasl.password | SASL password for use with the PLAIN and SASL-SCRAM |
 | protocol | security.protocol | Security protocol used to communicate with brokers |
 | sslKeyLocation | ssl.key.location | Path to client&#39;s private key (PEM) used for authentication |
 | sslKeyPassword | ssl.key.password | Password for client&#39;s certificate |
 | sslCertificateLocation | ssl.certificate.location | Path to client&#39;s certificate |
 | sslCaLocation | ssl.ca.location | Path to CA certificate file for verifying the broker&#39;s certificate |
```

Comment:
we can also validate & add if required the host.json setting for output

Reply:
Debug = kafkaOptions?.LibkafkaDebug,
                MetadataMaxAgeMs = kafkaOptions?.MetadataMaxAgeMs,
                SocketKeepaliveEnable = kafkaOptions?.SocketKeepaliveEnable

Reply:
Added